core/src/main/java/com/rudderstack/android/sdk/core/ResidencyServer.java

```diff
@@ -0,0 +1,6 @@
+package com.rudderstack.android.sdk.core;
+
+public enum ResidencyServer {
```

Comment:
How about renaming the term `ResidencyServer` to `DataResidencyServer` so that it gives more context about what exactly it's serving ?

Reply:
Also, I think it would be better to create a separate `Enums` class and store all the Enums it rather than each enum having its own class. 

In a similar notion of how we are grouping all the constants into a Constants class. 

If we want to go with this approach, we can even move the [NetworkResponses](https://github.com/rudderlabs/rudder-sdk-android/blob/2981bfefc991830d0cb2f6507119ed002d68b551/core/src/main/java/com/rudderstack/android/sdk/core/util/Utils.java#L237) as well

Reply:
Good idea will implement it.

Reply:
Renamed the class name to `DataResidencyServer`.